content/docs/2.4/scalers/rabbitmq-queue.md

```diff
@@ -101,6 +102,7 @@ spec:
       queueName: testqueue
       mode: QueueLength
       value: "20"
+      metricName: testqueue-1 #optional. Generated value would be `rabbitmq-host-testqueue`
```

Comment:
This feels odd. So it's more of a metric name suffix then?

Also, the -1 is missing - is that correct or not?

Reply:
@tomkerkhove I fixed the above example snippet.